dev/snippets/config/templates/README.md

```diff
@@ -97,6 +97,15 @@ follows:
   `stateful_widget_material`, except that it wraps the stateful widget with a
   `Scaffold`.
 
+- [`stateful_widget_center`](stateful_widget_center.tmpl) : Similar to
```

Comment:
I know it's longer, but maybe `stateful_widget_scaffold_center` would be a better description, since I can also see a use case for just having a center and not a scaffold, although we don't have that template yet, if we wanted to add it, then the obvious name would be taken.

Reply:
Done!